app.yaml

```diff
@@ -40,9 +40,17 @@ handlers:
     # WARNING TO DEVELOPERS: Files in this folder may be stale for
     # up to 10 mins after a code release.
     Cache-Control: 'public, max-age=600'
+- url: /assets/(constants\.json)
```

Comment:
Ah, I finally get it, but I think this line is fine and I need to change `'/assets/constants.json'` in [load_constants.js](https://github.com/oppia/oppia/pull/3033/files#diff-3554d6bb2da9e580f9d3b2a1bfe0adb4R17) to `GLOBALS.ASSET_DIR_PREFIX + '/assets/constants.json'`. This also mean we still need at least one `GLOBAL` because `ASSET_DIR_PREFIX` contains value returned by backend `utils.get_asset_dir_prefix()`

Alos, this additional rule is here just for longer `max-age`.